plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ModuleWrappingTransformation.xtend

```diff
@@ -256,24 +256,25 @@ class ModuleWrappingTransformation extends Transformation {
 
 				val module = state.info.getImportedModule(elementIM);
 
+				val isReactImport = JSXBackendHelper.isJsxBackendImportDeclaration(elementIM)
+
 				// calculate names in output
 				val completeModuleSpecifier =
-					if (JSXBackendHelper.isJsxBackendModule(module)) {
+					if (isReactImport) {
```

Comment:
There are other implementations of JSX, not just `React`.
Why change the name? This name suggests concrete implementation, instead intended semantics.

Reply:
See comment above